framework/src/main/java/org/checkerframework/framework/flow/CFAbstractStore.java

```diff
@@ -254,6 +255,46 @@ public void insertValue(FlowExpressions.Receiver r, AnnotationMirror a) {
         insertValue(r, analysis.createSingleAnnotationValue(a, r.getType()));
     }
 
+    /**
+     * Add the annotation {@code a} for the expression {@code r} (correctly deciding where to store
```

Comment:
The second parameter is called `newAnno`. So it would be better to either use that name here instead of `a` or change the parameter name to `a`.